MonstieWash/Assets/Scripts/Video/VideoController.cs

```diff
@@ -151,6 +160,13 @@ private void OnVideoFinish(VideoPlayer vp)
         StopVideo();
         status = Status.Idle;
         GameSceneManager.Instance.StartNewLevel(levelFollowingAnimatic);
+
+        // re-enable player input
+        InputSystem.EnableDevice(Keyboard.current);
+        InputSystem.EnableDevice(Mouse.current);
+        m_inputManager.Enable();
```

Comment:
Then you can just replace this part with the follwing to re-enable inputs:

```suggestion
        // re-enable player input
        InputManager.Instance.SetControlScheme(InputManager.ControlScheme.MenuActions);
```